Resources/doc/installation.rst

```diff
@@ -5,7 +5,7 @@ To install JMSTranslationBundle with Composer execute the following command:
 
 .. code-block :: bash
 
-    $ composer require jms/translation-bundle "~1.2.1"
+    $ composer require jms/translation-bundle "~1.3.2"

```

Comment:
In reality we should probably just suggest "^1.3" instead so we don't need to constantly update this until there's a major version update.

Reply:
Done :)